packages/dev-server-swc/demo/ts/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
is this necessary for the demo? the root gitignore should cover this

Reply:
The root gitignore doesn't seem to cover this.

```
koddsson@Kristjans-MBP dev-server-swc % git st
## swc-plugin
 D demo/ts/.gitignore
?? demo/ts/node_modules/
```

Note that I copied this over from `dev-server-esbuild` so the demo there also has this `.gitignore` file. 

https://github.com/modernweb-dev/web/blob/master/packages/dev-server-esbuild/demo/ts/.gitignore